brambling/views/utils.py

```diff
@@ -225,3 +228,66 @@ def get_context_data(self, **kwargs):
             'next_step': self.current_step.next_step if self.current_step else None,
         })
         return context
+
+
+class FinanceTable(object):
+
+    def __init__(self, event, transactions):
+        self.event = event
+        self.transactions = transactions
+
+    def headers(self):
```

Comment:
I think you could potentially replace `headers()` and `header_cells()` with a `headers` class attribute. Though if the idea is to provide space for flexibility down the line, that could be good. The convention would be `get_headers()` in that case.